src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAvrcpControl.cs

```diff
@@ -0,0 +1,415 @@
+        }
+
+        private void OnConnectionChanged(object s, AvrcpControlConnectionChangedEventArgs e)
+        {
+            if (e.RemoteAddress != RemoteAddress)
+            {
+                return;
+            }
+            if (_taskForConnection != null && !_taskForConnection.Task.IsCompleted)
+            {
+                if (e.IsConnected == true)
+                {
+                    _taskForConnection.SetResult(true);
+                    ConnectionStateChanged?.Invoke(this, e);
+                }
+                else
+                {
+                    _taskForConnection.SetException(BluetoothErrorFactory.CreateBluetoothException((int)BluetoothError.OperationFailed));
+                }
+                _taskForConnection = null;
+                return;
+            }
+            if (_taskForDisconnection != null && !_taskForDisconnection.Task.IsCompleted)
+            {
+                if (e.IsConnected == false)
+                {
+                    _taskForDisconnection.SetResult(true);
+                    ConnectionStateChanged?.Invoke(this, e);
+                }
+                else
+                {
+                    _taskForDisconnection.SetException(BluetoothErrorFactory.CreateBluetoothException((int)BluetoothError.OperationFailed));
+                }
+                _taskForDisconnection = null;
+            }
+        }
+
+        /// <summary>
+        /// The AvrcpControlConnectionChangedEventArgs event is invoked when the connection status of device is changed.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public event EventHandler<AvrcpControlConnectionChangedEventArgs> ConnectionStateChanged;
+
+        /// <summary>
+        /// The PositionChangedEventArgs event is invoked when the play position of a track is changed.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public event EventHandler<PositionChangedEventArgs> PositionChanged;
+
+        /// <summary>
+        /// The PlayStateChangedEventArgs event is invoked when the play state of a track gets changed.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public event EventHandler<PlayStateChangedEventArgs> PlayStateChanged;
+
+        /// <summary>
+        /// The TrackInfoChangedEventArgs event is invoked when info of a track gets changed.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public event EventHandler<TrackInfoChangedEventArgs> TrackInfoChanged;
+
+        /// <summary>
+        /// Asynchronously connects the remote device
+        /// </summary>
+        /// <exception cref="NotSupportedException">Thrown when the Bluetooth is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the method fails</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public Task ConnectAsync()
```

Comment:
How to set remote address to connect

Reply:
RemoteAddress is set on GetProfile() method.
BluetoothAvrcpControl instance call be obtained from this function.

        public T GetProfile<T>() where T : BluetoothProfile
        {
            // TODO : Need to check capability of supporting profiles
            var profile = (T)Activator.CreateInstance(typeof(T), true);
            profile.RemoteAddress = RemoteDeviceAddress;
            return profile;
        }

Reply:
Ok, I see, 
Is it impossible ?
``` c#
var device = BluetoothAdapter.GetBondedDevice(address1);
var device2 = BluetoothAdapter.GetBondedDevice(address2);
```

Reply:
It is possible.
each device info will be set on each variable device and device2.

Reply:
What's happen if I use another handle that is not connect, 
``` c#
var device1 = BluetoothAdapter.GetBondedDevice(address1);
var device2 = BluetoothAdapter.GetBondedDevice(address2);
var avrcpControl1 = device1.GetProfile<BluetoothAvrcpControl>();
var avrcpControl2 = device2.GetProfile<BluetoothAvrcpControl>();
await avrcpControl1.ConnectAsync();

// It seems update remote device of avrcpControl1
avrcpControl2.RepeatMode = RepeatMode.AllTrack;
```

Reply:
InvalidOperationException("Remote device not connected") will be occured.

Reply:
Why, `avrcpControl1` and `avrcpControl2` are sharing same internal object
```
        public RepeatMode RepeatMode
        {
            get
            {
                return BluetoothAvrcpControlImpl.Instance.GetRepeatMode();
            }
            set
            {
                BluetoothAvrcpControlImpl.Instance.SetRepeatMode(value);
            }
        }
```

Reply:
Sorry, what I said is wrong.
avrcpControl1.RepeatMode is working same with avrcpControl2.RepeatMode..

So, the answer for previous question should be fixed like below:
What's happen if I use another object that is not connect,
> That also works.. I mean avrcpControl.RepeatMode can be set by using avrcpControl2 because this function is for only connected device.

Reply:
So, I want to add a new property to check connection state on BluetoothAvrcpControl class
this property will be blocking the execution on disconnected state

Reply:
Do you mean adding a new property like below?
In my thought,, it is fine to return the correct state from another class(avrcpControl2) because the state is global state.

get
            {
                if (isConnected == true)
                {
                    return BluetoothAvrcpControlImpl.Instance.GetEqualizerState();
                }   
                else
                {
                    BluetoothErrorFactory.ThrowBluetoothException((int)BluetoothError.OperationFailed);
                    return EqualizerState.Off;
                }   
            }

Reply:
I don't agree, it seems not global properties, it is properties of connected peer device
If connected peer devices was changed, properties also will be changed.